### DISCUSSION TITLE
How will i get the Galaxy Brain badge on my profile section ASAP

### DISCUSSION THREAD

**Discussion by jagannathVittal** [community/community#58865] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can i get the Galaxy Brain badge on my profile section easily

**Comment by jees47** [comment, marked as answer]
By 2 accepted answers

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
